### PR TITLE
Additional repository setup

### PR DESCRIPTION
**Pull request by alexander-demichev**
Add `make unit|images|push` tasks, dockerfiles and a task for verifying vendor content

### CODE REVIEW

**Review by elmiko — COMMENTED** [review]
this generally looks good to me, i'm not so familiar with the envtest stuff though.
/lgtm

**Comment by alexander-demichev** [comment]
@elmiko I copied envtest setup from cccmo

**Comment by lobziik** [comment]
Hm... @alexander-demichev can you please explain, why you want two different dockerfiles there?

File: Makefile [hunk 1]
```diff
@@ -1,16 +1,23 @@
+IMG ?= controller:latest
+
 all: build
 
-verify: fmt vet 
+verify-%:
+	make $*
+	./hack/verify-diff.sh
```
Comment (JoelSpeed) [inline Makefile:7]: Just a note that in the CI, I think we need to run `verify-fmt` and `verify-vet` to make sure that once those tools have run, there's no diff left over

**Comment by elmiko** [comment]
> Hm... @alexander-demichev can you please explain, why you want two different dockerfiles there?

iiuc, the `Dockerfile` allows community build, whereas `Dockerfile.rhel` is restricted to people with RHEL subscription or our CI systems.

**Comment by alexander-demichev** [comment]
@elmiko you are right, dockerfiles reference different images. One is publicly available for anyone outside of red hat, second is rhel based

**Pull request merged**